pkg/names/names.go

```diff
@@ -14,4 +14,8 @@ const LEADER_LABEL = "kubemacpool-leader"
 
 const LEADER_ID = "kubemacpool-election"
 
-const ADMISSION_IGNORE_LABEL = "kubemacpool/ignoreAdmission"
+const K8S_RUNLABEL = "runlevel"
+
+const OPENSHIFT_RUNLABEL = "openshift.io/run-level"
+
+var RUNLABEL = []string{"1", "0"}
```

Comment:
Why do we need this? It hurts my brain that `RUNLABEL[0] == "1"` and `RUNLABEL[1] == "0"`.

Reply:
I change it to be 0 and 1.

this variables are using by openshift and k8s (0 is critical namespace and 1 is infrastructure namespace)